adventurous-anteaters/README.md

```diff
@@ -44,3 +44,33 @@
 - Time Management (yeah, right!)
 
 [WE EVEN MADE A SHEET](https://docs.google.com/spreadsheets/d/1txXqRkj4V4D4RS1sVEganfGJPK0br0lnWWaH3P1boUM/edit#gid=873896706)
+
+
+
+### Credits:
+#### The assets came from these MIT licensed repos on Github: 
+[Packard Belle: A nostalgic component UI](https://github.com/padraigfl/packard-belle)
+[98.css](https://github.com/jdan/98.css)
+
+
+### Licensing Information
+
+Copyright (c) 2020 @ayushxx7
```

Comment:
Sorry, but you can't do this. This project will be licensed by the license that's in the root folder, and the copyright will be to Python Discord. It's still MIT licensed by us, but we're not gonna let the teams copyright claim their work because it adds a lot of potential micromanagement for us.

Please remove this license from the readme altogether.